.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest ]
-        python-version: [ '3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.9', '3.10', '3.11']
         pydantic-version: [ '1.10.15', '2.7.1' ]
```

Comment:
not related to this PR, do we still support pydantic v1?

Reply:
I was thinking to remove it in a future PR. Was about to ask you about that.